frontend/src/pages/MobileCycleCountPage.jsx

```
@@ -0,0 +1,233 @@
+  const [activeTab, setActiveTab] = useState('in_progress');
+  const [showCompleted, setShowCompleted] = useState(false);
+  
+  useEffect(() => {
+    dispatch(fetchCycleCountBatches());
+    dispatch(fetchCycleCountStats());
+  }, [dispatch]);
+  
+  // Filter batches based on search term and active tab
+  const filteredBatches = batches ? batches.filter(batch => {
+    const searchMatch = !searchTerm || 
+      (batch.name && batch.name.toLowerCase().includes(searchTerm.toLowerCase()));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)